### DISCUSSION TITLE
Cydia version reports needing IOS 14 despite all webpages saying it will run on 11

### DISCUSSION THREAD

**Discussion by EzioisAwesome56** [utmapp/UTM#6490] [UTM Help (iOS)] [answered]
**Describe the issue**
Trying to download the UTM app from cydia says it requires IOS 14, while all related webpages says it will run on iOS 11

**Configuration (required)**
* UTM Version: N/A latest
* OS Version: iOS 12.5.7
* Device Model: iPad Air 1
* Is it jailbroken (name jailbreak used)? yes, Checkra1n jailbreak
* How did you install UTM? N/A

I'm aware anything probably won't run well on this device, but I think it would still be fun to try it out anyway
![20240715_205629](https://github.com/user-attachments/assets/8a45a34d-9794-412a-877d-05015c4cfb97)
![image](https://github.com/user-attachments/assets/37862095-11da-48fe-bc24-05d96f03006e)
![image](https://github.com/user-attachments/assets/81fba2e6-3354-40f3-9bfa-9cf92a19c00b)
![20240715_205634](https://github.com/user-attachments/assets/9a474560-08c7-49a9-a837-146f805c5c97)

**Comment by osy** [comment, marked as answer]
https://docs.getutm.app/installation/ios/

See the note

> The last version of UTM that supports iOS 11-13 is [v3.2.4](https://github.com/utmapp/UTM/releases/tag/v3.2.4).

**Reply by EzioisAwesome56** [reply]
i somehow missed this note, thanks for pointing it out!
